pdns/ws-auth.cc

```diff
@@ -826,6 +810,22 @@ static void updateDomainSettingsFromDocument(UeberBackend& B, const DomainInfo&
         }
       }
     }
+
+    // Rectify
```

Comment:
```suggestion
    // Rectify after ALL RRset modifications are done (incl. SOA)
```

Reply:
The proposed comment is confusing. Rectify is also triggered by non rrset changes here.

Reply:
good point, in fact my test case was changing nsec3param!

Reply:
```suggestion
    // Rectify. This needs to happen after all (RRset and other) modification are done, incl. automatic RRs.
```